examples/SCAMPy/src/helper_funcs.jl

```diff
@@ -8,13 +8,20 @@ using JSON
 using Random
 # EKP modules
 using EnsembleKalmanProcesses.ParameterDistributionStorage
+# TurbulenceConvection.jl
+using TurbulenceConvection
+tc_dir = dirname(dirname(pathof(TurbulenceConvection)));
+include(joinpath(tc_dir, "integration_tests", "utils", "main.jl"))
+
+
+data_directory(root::S, name::S, suffix::S="00000") where S<:AbstractString = joinpath(root, "Output.$name.$suffix")
+namelist_directory(root::S, casename::S) where S<:AbstractString = joinpath(root, "namelist_$casename.in")
 
 """
-    run_SCAMPy(
+    run_SCM(
```

Comment:
For the time being, calibration of TC.jl should be an addition to the codebase, not a substitution of the functions to calibrate SCAMPy. Please update this PR to ensure backward compatibility and allow testing the reproducibility of results of all users.